debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java

```diff
@@ -244,6 +245,16 @@ private Object convertToBoolean(String value) {
         }
     }
 
+    private DateTimeFormatter dateWithOptionalTimeFormatter() {
```

Comment:
Can you convert this to a statically defined attribute as it is unnecessary to rebuild it every time the converison is needed.